src/desktop/apps/artist/server.tsx

```diff
@@ -90,7 +90,13 @@ app.get(
       res.status(status).send(layout)
     } catch (error) {
       console.log(error)
-      next(error)
+      if (error.message.includes("Artist Not Found")) {
```

Comment:
Theoretically there could be _other_ errors here, of which 'Artist Not Found' is only one.

However, this is the error handler for the server-side route handler, so we're definitely not serving a 2xx page already at this point. So assuming its a 404 and returning 'Artist Not Found' is _probably_ semi reasonable, especially if it's just a quick fix while a better one is worked out in Reaction.